python-worker/src/al_iyaal_worker/moderation/llm.py

```
@@ -0,0 +1,301 @@
+    return enriched
+
+
+def _post_json(
+    url: str,
+    payload: dict[str, Any],
+    headers: dict[str, str],
+    timeout_seconds: float = 600.0,
+) -> dict[str, Any]:
+    body = json.dumps(payload).encode("utf-8")
+    request_headers = {"Content-Type": "application/json", **headers}
+    http_request = request.Request(url, data=body, headers=request_headers, method="POST")
```

Ruff (0.15.5)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)